loader/include/Geode/utils/web.hpp

```diff
@@ -122,12 +134,20 @@ namespace geode::utils::web {
 
         WebRequest& userAgent(std::string_view name);
 
+        WebRequest& encoding(std::string_view encoding);

```

Comment:
encoding of what? the request or the response? and what are valid values?

Reply:
[its the encoding of the response](https://curl.se/libcurl/c/CURLOPT_ACCEPT_ENCODING.html), specifically a download response. as for the valid values, it depends on libcurl, but an example of valid values are br, gzip, deflate. these can be passed to encoding as a comma separated string.